application/app/Models/User.php

```diff
@@ -56,4 +56,10 @@ public function tasks()
     {
         return $this->hasMany(Task::class, 'created_user_id');
     }
+
+    //ユーザーのコメントを取得
+    public function commenter()
```

Comment:
```suggestion
    public function comment()
```
コメントした人は自分なので、commentが適切かと思います。